app/controllers/OneOffContributions.scala

```diff
@@ -31,12 +32,12 @@ class OneOffContributions(
 
   implicit val ar = assets
 
-  def displayForm: Action[AnyContent] = CachedAction() {
-    form(uatMode = false)
+  def displayForm(paypal: Option[Boolean] = Some(false)): Action[AnyContent] = CachedAction() {
```

Comment:
You don't need the default value `Some(false)` because line 56 already makes the default false